### PR TITLE
feature labels for cards

### PR DESCRIPTION
**Pull request by 3elk4**
+ create model for labels
+ create junction table for labels and cards
+ mutations updated
+ connections in models updated
+ get card's labels
+ assign and unassign label to card
+ get all labels
+ routes updated
+ seeds updated

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -31,11 +31,20 @@
...
   post 'edit_card', to: 'card#edit'
   post 'archive_card', to: 'card#archive'
   post 'restore_card', to: 'card#restore'
 
+  #label CRUD
+  get 'get_labels', to: 'label#get_all'
+
+  #card-label CRUD
+
+  post 'assign_label/:card_id/:label_id', to: 'cardlabel#assign_label'
```
Comment (BigDinDonMan) [inline config/routes.rb:45]: we do not really need id's in endpoints if request method is POST, we can pass them in the body

**Review by BigDinDonMan — CHANGES_REQUESTED** [review]
🔥 🔥 🔥 
Great job, just change the id's in endpoints and its fine

File: config/routes.rb (hunk 1 above)
Comment (3elk4) [reply config/routes.rb]: Sure, let's change this 🚀

**Review by BigDinDonMan — APPROVED** [review]
Gr8 job m8!

**Review by BigDinDonMan — APPROVED** [review]

**Pull request merged**